ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_l3_edge_l3_interfaces.schema.yml

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+$defs:
+  l3_edge_l3_interfaces:
+    type: list
+    items:
+      type: dict
+      keys:
+        speed:
+          type: str
+          description: |-
+            Speed should be set in the format `<interface_speed>` or `forced <interface_speed>` or `auto <interface_speed>`.
+        ip:
+          type: str
+          description: Node IPv4 address/Mask or 'dhcp'.
+        dhcp_client_accept_default_route:
+          type: bool
+          description: |-
+            Supported if `ip` is `dhcp`.
+            Accepts default route from DHCP, default False.
```

Comment:
If default is false just set that in the schema instead of writing it :)

Reply:
sure .. though it has the same effect given that we do nothing with it today :)